extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -1153,6 +1158,37 @@ export class Bind {
     return null;
   }
 
+  /**
+   * Hopefully we can delete this with Safari 13+.
+   * @param {!Element} element
+   * @param {string} property
+   * @param {BindExpressionResultDef} newValue
+   */
+  updateSelectForSafari_(element, property, newValue) {
+    // We only care about option[selected].
+    if (element.tagName !== 'OPTION' || property !== 'selected') {
+      return;
+    }
+    // We only care if this option was selected, not deselected.
+    if (!newValue) {
+      return;
+    }
+    // Workaround only needed for Safari.
+    if (!Services.platformFor(this.win_).isSafari()) {
```

Comment:
nit. closure compiler will group these boolean conditions together but it seems logical to me that this should be the first statement in the checks. Return immediately if is not safari, rather then checking the element properties and then reaching this statement.

Reply:
I considered this but thought it might be marginally cheaper to do string equality vs. function call.